daffodil-tdml-lib/src/main/scala/org/apache/daffodil/tdml/TDMLRunner.scala

```diff
@@ -634,15 +642,33 @@ abstract class TestCase(testCaseXML: NodeSeq, val parent: DFDLTestSuite) {
   lazy val optExpectedOrInputInfoset = (testCaseXML \ "infoset").headOption.map { node =>
     new Infoset(node, this)
   }
-  lazy val optExpectedErrors: Option[ExpectedErrors] = (testCaseXML \ "errors").headOption.map {
-    node => ExpectedErrors(node, this)
+  lazy val optExpectedErrors: Option[Seq[ExpectedErrors]] = {
+    val e = (testCaseXML \ "errors" \\ "error")
```

Comment:
I'm of the mind to have them all behave consistently. So if we have ` <tdml:validationErrors />` mean that there should be no validation errors, it should be the same for `<tdml:errors />` and `<tdml:warnings />`. But if we have that functionality, then I don't think we need the noWarnings/noValidationErrors attributes. Thoughts?

Reply:
The problem if we didn't support `noWarnings` is it would be a huge effort to add `<tdml:warnings />` to all tests to ensure warnings are never  ignored. We eventually want get to the point where no tests have unexpected warnings and your new property is definitely the preferred way to enforce that globally over `<tdml:warnings />` which can't be added globally.

If we do want consistency with validation, we may want to consider deprecating `<tdml:validation />` and replacing it with a property similar to the noWarnings one . Just like with warnings, we probably want to get the the point where there are no unexpected validation errors, and it's much easier to enforce that by flipping a default flag than adding `<tdml:validaiton />` to all the tests. If we do this, I'd suggest it be a separate PR since there's backwards compatibility issues to consider.

Also, I don't think it's worth the added complexity right now to support something like `<tdml:errors />` means "some errors exist but I don't care about the message". That might be useful at some point, but that could also be a separate PR if we want it. The key thing we really need is the noWarnings capability.

Reply:
Some more thoughts. I would suggest we make it so `<tdml:warnings />` and `<tdml:errors />` is not allowed--each one must have child elements. That keeps things simple an intuitive. We don't really need a way to say "there must be an error/warning, but I don't care what it is"--that just isn't really that necessary and adds complication. 

For now, we still allow `<tdml:validation />` for backwards compatibility, but we should eventually plan to deprecate that in favor of a new property similar to the "no warnings" property. And when we add that property, we can support backwards compatibility by making it so `<tdml:validationErrors />` enables the "noValidationErrors" flag, but is otherwise ignored. Eventually we can drop that backwards compatibility and make `<tdml:validationErrors />` invalid to match `<tdml:warnings />` and <tdml:errors />`. These changes to how validationErrors works can all be done in future PRs though. It's fine if there is a slight inconsistency for now.